libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Orchestrator;
+
+namespace Microsoft.Bot.Builder.AI.Orchestrator
+{
+    public class OrchestratorRecognizer : IRecognizer
```

Comment:
>OrchestratorRecognizer [](start = 17, length = 22)

Why?  What is the rationale?  IS there something about the OrchestratorRecognizer that isn't 100% overlap with this?  We should NOT propagate the stink of LuisREcognizer code base into recognizers in general, it's an anti-pattern.  There is no reason to do this.

Reply:
Updated.